jdiagram/bnd.bnd

```diff
@@ -0,0 +1,49 @@
+Bundle-SymbolicName: jdiagram;singleton:=true
+Bundle-Name: jdiagram
+Bundle-Version: 1.0.0.0
+Bundle-ManifestVersion: 2.0
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ActivationPolicy: lazy
+Bundle-Activator: gov.ca.dwr.jdiagram.Activator
+Bundle-Vendor: California Department of Water Resources
+-noee true
+
+Require-Bundle: \
+    org.eclipse.ui,\
+    org.eclipse.core.runtime,\
+    org.eclipse.core.resources,\
+    org.eclipse.ui.navigator,\
+    org.eclipse.ui.navigator.resources,\
+    wrims-ide;bundle-version="1.0.0";visibility:=reexport,\
+    hec-dependencies;bundle-version="1.0.0"
+
+Import-Package: \
+    com.mindfusion.drawing,\
+    gov.ca.dwr.hecdssvue,\
+    gov.ca.dwr.hecdssvue.panel,\
+    gov.ca.dwr.hecdssvue.views,\
+    wrimsv2_plugin.debugger.core,\
+    wrimsv2_plugin.debugger.exception,\
+    wrimsv2_plugin.tools
+
+-exportcontents: \
+    com.mindfusion.common,\
+    com.mindfusion.diagramming,\
+    com.mindfusion.diagramming.export,\
+    com.mindfusion.diagramming.jlayout,\
+    com.mindfusion.diagramming.lanes,\
+    com.mindfusion.drawing,\
+    com.mindfusion.graphs,\
+    com.mindfusion.pdf,\
+    com.mindfusion.svg
+
+Export-Package: \
+    gov.ca.dwr.jdiagram,\
+    gov.ca.dwr.jdiagram.views
+
+-includeresource: \
+  lib_x64/JDiagram.jar=../third-party/libs/JDiagram.jar;lib:=true,\
```

Comment:
I'm confused why this would need to include a jar, isn't that exported from third-party?

Reply:
I attempted to use the JDiagram jar through the third-party module and it caused runtime exceptions:
java.lang.NoClassDefFoundError: Could not initialize class javax.xml.bind.DatatypeConverterImpl
Since it's only 1 jar, bundling the JDiagram jar directly in the jdiagram module.

Reply:
I would argue that needs to be accomplished under this milestone as it breaks the design of the third-party module.